pkg/operator/ceph/cluster/cluster_cleanup_test.go

```diff
@@ -0,0 +1,48 @@
+package cluster
+
+import (
```

Comment:
In addition to this unit test, we should really do this cleanup as part of an integration test. I'm thinking at the end of the helm suite since it's one of the shorter tests. For example:
- Pass a new flag to `Teardown()` on [this line](https://github.com/rook/rook/blob/master/tests/integration/ceph_helm_test.go#L77) of the helm test. 
- Inside the `UninstallRookFromMultipleNS()` method, if that flag is set:
   - Set the cleanup flag on the CR
   - Wait for the jobs to start
   - Verify that a job for the node(s) starts and completes successfully.